app/assets/javascripts/controllers/container_group/container_group_controller.js

```diff
@@ -0,0 +1,29 @@
+angular.module('containerGroupDashboard', ['ui.bootstrap', 'patternfly', 'patternfly.charts', 'containerDashboard'])
```

Comment:
Could we please keep the style consistent? All the other files use 2 spaces for indent, please fix the new ones to match that.